theme.json

```
@@ -66,10 +96,17 @@
         "appearanceTools": true,
         "useRootPaddingAwareAlignments": true,
         "layout": {
-            "contentSize": "1440px",
-            "wideSize": "1440px"
+            "contentSize": "var(--dswp-content-size)",
+            "wideSize": "var(--dswp-content-size)"
         },
         "typography": {
+            "fontFamilies": [
+                {
+                    "fontFamily": "BCSans",
+                    "name": "BC Sans",
+                    "slug": "bcsans-font-family"
```

```suggestion
                    "slug": "bcsans"
```
The class that it produced on blocks was `has-bcsans-font-family-font-family`, so this slug should produce `has-bcsans-font-family`.